### PR TITLE
Javadoc and OSGi

### PR DESCRIPTION
**Pull request by agudian**
That's two small build related issues in one pull-request:

* #805 use basic ASCII chars in the javadoc ASCII art tables (I didn't find a proper way to get it to work otherwise...)
* #27 Add OSGi metadata to manifests of mapstruct.jar and mapstruct-jdk8.jar

### CODE REVIEW

File: parent/pom.xml [hunk 1]
```diff
@@ -329,7 +329,20 @@
...
+                <plugin>
+                    <groupId>org.apache.felix</groupId>
+                    <artifactId>maven-bundle-plugin</artifactId>
+                    <version>3.0.1</version>
+                    <executions>
+                        <execution>
+                            <id>bundle-manifest</id>
+                            <phase>process-classes</phase>
+                            <goals>
+                                <goal>manifest</goal>
```
Comment (sjaakd) [inline parent/pom.xml]: So, what will be added to the manifest with this plugin?

**Comment by sjaakd** [comment]
One question. For remainder ok. Please go ahead and merge.

File: parent/pom.xml (hunk 1 above)
Comment (gunnarmorling) [inline parent/pom.xml]: It will contain the required OSGi headers, like exported packages, version etc.

**Comment by gunnarmorling** [comment]
Looks good :)

**Comment by agudian** [comment]
Thanks for checking, guys!

**Pull request merged**